password_security/tests/test_res_users.py

```diff
@@ -16,6 +16,7 @@ class TestResUsers(SavepointCase):
     def setUp(cls):
         super(TestResUsers, cls).setUp()
         cls.main_comp = cls.env.ref('base.main_company')
+        cls.main_comp.password_no_login = True
```

Comment:
Put this (suitable adapted, use self instead of cls) in the test method instead of in the setup. In stead of the assertion on the value. No need to reset it in the teardown, because change will automatically be rolled back.